src/main/java/com/josdem/vetlog/model/Vaccination.java

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2024 Jose Morales contact@josdem.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.josdem.vetlog.model;
+
+import static jakarta.persistence.EnumType.STRING;
+import static jakarta.persistence.GenerationType.IDENTITY;
+
+import com.josdem.vetlog.enums.PetType;
```

Comment:
Also:
- Seems like we are not using `PetType` here, please remove.
- We normally avoid to use wildcard imports to avoid class name collitions, here is a way to remove them using IntelliJ: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

Reply:
Fixed